_posts/hdubbe/2021-11-24-project-update

```diff
@@ -0,0 +1,10 @@
+---
+layout: Project Update - Thanksgiving Break
```

Comment:
This update post is looking good! I think you will want to set the layout in the header here to as `layout: post`.